worlds/oot/Rules.py

```diff
@@ -223,7 +223,7 @@ def set_shop_rules(ootworld):
 # The goal is to automatically set item rules based on age requirements in case entrances were shuffled
 def set_entrances_based_rules(ootworld):
 
-    if ootworld.multiworld.accessibility == 'beatable': 
+    if ootworld.multiworld.accessibility[ootworld.player] == 'minimal':
```

Comment:
```py
    if ootworld.multiworld.accessibility == 'beatable':
```
looks like a condition that can never be fulfilled, so this change would make it possible to enter a code path that is currently unreachable. Which might be desirable and fix a bug, or it might just expose other unintended behaviors, I don't know. But better to get more context.

Reply:
https://discord.com/channels/731205301247803413/731214280439103580/1161465932959469638